operations/_scripts/generate/terraform/generate_vpc_tf.sh

```diff
@@ -0,0 +1,26 @@
+#! /bin/bash
+echo "In generate_vpc_tf.sh"
+
+if [[ $CREATE_VPC == true ]]; then
+vpc_block="resource \"aws_vpc\" \"main\" {
+ count = var.create_vpc == \"true\" ? 1 : 0
```

Comment:
if these counts are using the `create_vpc` var in TF already, what's the benefit of using the generator script instead of just having `.tf` files directly in the TF?